src/service/resources-impl.js

```diff
@@ -1049,7 +1068,14 @@ export class Resources {
       this.schedule_(resource,
           LAYOUT_TASK_ID_, LAYOUT_TASK_OFFSET_,
           opt_parentPriority || 0,
-          resource.startLayout.bind(resource));
+          (isDocumentVisible) => {
```

Comment:
I don't think this is a good place. I think a better place would be in `taskComplete_`.